docs/install/install.rst

```diff
@@ -336,4 +336,63 @@ issues locating the installed libraries:
 .. note::
 
    If ROCm support is enabled, you might have to add the path to the ROCm libraries to ``LD_LIBRARY_PATH``,
-   for example, ``export LD_LIBRARY_PATH=/opt/rocm/lib:${LD_LIBRARY_PATH}``.
\ No newline at end of file
+   for example, ``export LD_LIBRARY_PATH=/opt/rocm/lib:${LD_LIBRARY_PATH}``.
+
+Post-installation troubleshooting
+========================================
+
+This section explains how to resolve certain issues that might happen when you first use Omnitrace.
+
+Issues with RHEL and SELinux
+----------------------------------------------------
+
+RHEL (Red Hat Enterprise Linux) distributions of Linux automatically enable a security feature 
+named SELinux (Security-Enhanced Linux) that prevents Omnitrace from running.
+This issue applies to systems running the CentOS 9 operating system using
+any version of the AMD ROCm Software alongside an AMD Instinct MI300 GPU.
+
+The problem occurs after you instrument a program and try to
+run ``omnitrace-run`` with the instrumented program.
+
+.. code-block:: shell
+
+   g++ hello.cpp -o hello
+   omniperf-instrument -M sampling -o hello.instr -- ./hello
+   omnitrace-run -- ./hello.instr
+
+Instead of successfully running the binary with call-stack sampling, 
+Omnitrace crashes with a segmentation fault.
```

Comment:
It maybe worth noting, that if you are physically on the system (as in, not SSH'ed in), your OS will probably show a pop-up warning from SELinux in the notifications area.

Reply:
I'll add this as a note.